client/src/app/site/pages/meetings/pages/agenda/modules/topics/modules/topic-poll/components/topic-poll-meta-info/topic-poll-meta-info.component.html

```diff
@@ -24,7 +24,7 @@
 
             <span *ngFor="let group of poll.entitled_groups; let last = last">
                 <span>{{ group.getTitle() | translate }}</span>
-                <span>{{ !last ? ', ' : '' }}</span>
+                <span *ngIf="!last">,&nbsp;</span>
```

Comment:
How would you like this code to look then?

Reply:
Just tested it and simply using a normal whitespace works. Like that:
```suggestion
                <span *ngIf="!last">, </span>
```

Reply:
That one will be removed by the cleanup script since the linter doesn't allow it.
Using `{{ !last ? ', ' : '' }}` would work, but @bastianjoel already asked not to do that.

Reply:
Use `&ensp;` then.

Reply:
> That one will be removed by the cleanup script since the linter doesn't allow it. 

Ah, I didn't know that! Do you know which rule disallows this? A quick google search did not return anything... Anyway, what @bastianjoel says.